service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocumentGeneratorServiceTest.java

```diff
@@ -27,19 +29,21 @@
     @Test
     void testEmptyCaseDetailsSuccessfullyReturnsByteArray() throws IOException {
         CaseDetails caseDetails = emptyCaseDetails();
-
         String content = textContentOf(documentGeneratorService.generateSubmittedFormPDF(caseDetails));
 
-        assertThat(content).contains("Case ID: 123");
+        assertThat(content).contains("C110A");
     }
 
+    @Disabled
     @Test
     void testPopulatedCaseDetailsSuccessfullyReturnsByteArray() throws IOException {
```

Comment:
you need to rename this test then

Reply:
I think @geek-soft wanted to do some work on this. Damian?

Reply:
Yes, I am working on this to reenable it and make it more stable.